%!TEX TS-program = xelatex
%!TEX encoding = UTF-8 Unicode
% Awesome CV LaTeX Template for CV/Resume
%
% This template has been downloaded from:
% https://github.com/posquit0/Awesome-CV
%
% Author:
% Claud D. Park <posquit0.bj@gmail.com>
% http://www.posquit0.com
%
%
% Adapted to be an Rmarkdown template by Mitchell O'Hara-Wild
% 23 November 2018
%
% Template license:
% CC BY-SA 4.0 (https://creativecommons.org/licenses/by-sa/4.0/)
%
%-------------------------------------------------------------------------------
% CONFIGURATIONS
%-------------------------------------------------------------------------------
% A4 paper size by default, use 'letterpaper' for US letter
\documentclass[11pt,a4paper,]{awesome-cv}

% Configure page margins with geometry
\usepackage{geometry}
\geometry{left=1.4cm, top=.8cm, right=1.4cm, bottom=1.8cm, footskip=.5cm}


% Specify the location of the included fonts
\fontdir[fonts/]

% Color for highlights
% Awesome Colors: awesome-emerald, awesome-skyblue, awesome-red, awesome-pink, awesome-orange
%                 awesome-nephritis, awesome-concrete, awesome-darknight

\definecolor{awesome}{HTML}{990000}

% Colors for text
% Uncomment if you would like to specify your own color
% \definecolor{darktext}{HTML}{414141}
% \definecolor{text}{HTML}{333333}
% \definecolor{graytext}{HTML}{5D5D5D}
% \definecolor{lighttext}{HTML}{999999}

% Set false if you don't want to highlight section with awesome color
\setbool{acvSectionColorHighlight}{true}

% If you would like to change the social information separator from a pipe (|) to something else
\renewcommand{\acvHeaderSocialSep}{\quad\textbar\quad}

\def\endfirstpage{\newpage}

%-------------------------------------------------------------------------------
%	PERSONAL INFORMATION
%	Comment any of the lines below if they are not required
%-------------------------------------------------------------------------------
% Available options: circle|rectangle,edge/noedge,left/right

\photo{./img/me.jpg}
\name{Filip}{Horvat}

\position{Bioinformatician}
\address{Institute of Molecular Genetics, Prague, Czech Republic}

\email{\href{mailto:fihorvat@gmail.com}{\nolinkurl{fihorvat@gmail.com}}}
\orcid{0000-0002-1896-7645}
\googlescholar{aA\_cIdYAAAAJ}
\github{fhorvat}
\linkedin{filip-horvat-6a8538b2}

% \gitlab{gitlab-id}
% \stackoverflow{SO-id}{SO-name}
% \skype{skype-id}
% \reddit{reddit-id}


\usepackage{booktabs}

\providecommand{\tightlist}{%
	\setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}

%------------------------------------------------------------------------------


\usepackage{float}
\usepackage{multicol}
\usepackage{colortbl}
\arrayrulecolor{white}
\usepackage{hhline}
\definecolor{light-gray}{gray}{0.95}

% Pandoc CSL macros
\newlength{\cslhangindent}
\setlength{\cslhangindent}{1.5em}
\newlength{\csllabelwidth}
\setlength{\csllabelwidth}{2em}
\newenvironment{CSLReferences}[3] % #1 hanging-ident, #2 entry spacing
 {% don't indent paragraphs
  \setlength{\parindent}{0pt}
  % turn on hanging indent if param 1 is 1
  \ifodd #1 \everypar{\setlength{\hangindent}{\cslhangindent}}\ignorespaces\fi
  % set entry spacing
  \ifnum #2 > 0
  \setlength{\parskip}{#2\baselineskip}
  \fi
 }%
 {}
\usepackage{calc}
\newcommand{\CSLBlock}[1]{#1\hfill\break}
\newcommand{\CSLLeftMargin}[1]{\parbox[t]{\csllabelwidth}{\honortitlestyle{#1}}}
\newcommand{\CSLRightInline}[1]{\parbox[t]{\linewidth - \csllabelwidth}{\honordatestyle{#1}}}
\newcommand{\CSLIndent}[1]{\hspace{\cslhangindent}#1}

\begin{document}

% Print the header with above personal informations
% Give optional argument to change alignment(C: center, L: left, R: right)
\makecvheader

% Print the footer with 3 arguments(<left>, <center>, <right>)
% Leave any of these blank if they are not needed
% 2019-02-14 Chris Umphlett - add flexibility to the document name in footer, rather than have it be static Curriculum Vitae


%-------------------------------------------------------------------------------
%	CV/RESUME CONTENT
%	Each section is imported separately, open each file in turn to modify content
%------------------------------------------------------------------------------



\hypertarget{professional-profile}{%
\section{Professional Profile}\label{professional-profile}}

Highly skilled and experienced computational biologist finishing a PhD
in the field. Strong expertise in developing and implementing
computational methods for analyzing complex biological data, including
genomic and transcriptomic data. Proficient in programming languages
such as R and Bash, with experience working in Linux environments.
Extensive experience collaborating with multidisciplinary and
international teams of scientists on elucidating mechanisms of various
biological processes. Seeking a challenging role in industry where my
skills and experience can be leveraged to advance scientific discovery
and bridge the gap between the computational and experimental research.

\hypertarget{professional-experience}{%
\section{Professional Experience}\label{professional-experience}}

\begin{cventries}
    \cventry{PhD student - computational research}{Institute of Molecular Genetics, Laboratory of Epigenetic Regulations}{Prague}{2017 - current}{\begin{cvitems}
\item Computational analysis and integration of high-throughput datasets
\item Conducting highly customized analyses and visualizations of complex biological data in R
\item Developing pipelines that automate processing of next generation sequencing datasets generated in the lab
\item Bioinformatic contribution to 10 high-level publications
\end{cvitems}}
    \cventry{Specialist of science and research - computational research}{Institute of Molecular Genetics, Laboratory of Epigenetic Regulations}{Prague}{2016 - 2017}{\begin{cvitems}
\item Statistical and differential expression analysis of RNA-seq datasets
\end{cvitems}}
    \cventry{Intern - computational research}{University of Zagreb, Faculty of Science, Bioinformatics Group}{Zagreb}{2015 - 2016}{\begin{cvitems}
\item Management, interpretation, visualization, and statistical analysis of high-throughput sequencing data
\end{cvitems}}
    \cventry{Intern - laboratory research}{Max F Perutz Laboratories, DNA Damage Response and Transcription Regulation}{Vienna}{Apr 2015 - Sep 2015}{\begin{cvitems}
\item Hands-on experience with various techniques such as PCR, gel electrophoresis, cell culture and DNA extraction
\end{cvitems}}
\end{cventries}

\hypertarget{education}{%
\section{Education}\label{education}}

\begin{cventries}
    \cventry{PhD in Developmental and Cell Biology}{Charles University, Faculty of Science}{Prague}{2017 - current}{}\vspace{-4.0mm}
    \cventry{Master in Molecular Biology}{University of Zagreb, Faculty of Science}{Zagreb}{2013 - 2016}{}\vspace{-4.0mm}
    \cventry{Bachelor in Molecular Biology}{University of Zagreb, Faculty of Science}{Zagreb}{2009 - 2013}{}\vspace{-4.0mm}
\end{cventries}

\pagebreak

\hypertarget{skills-and-qualifications}{%
\section{Skills and Qualifications}\label{skills-and-qualifications}}

\begin{cventries}
    \cventry{R (advanced), Bash (intermediate), Python (beginner)}{Programming languages}{}{}{}\vspace{-4.0mm}
    \cventry{Shiny, RMarkdown, Bioconductor, tidyverse, ggplot2, plotly}{R framework}{}{}{}\vspace{-4.0mm}
    \cventry{Unix/Linux, Command line interface, Nextflow, Git}{Other computational tools}{}{}{}\vspace{-4.0mm}
    \cventry{English (fluent), German (basic), Croatian (native)}{Languages}{}{}{}\vspace{-4.0mm}
\end{cventries}

\hypertarget{teaching-experience}{%
\section{Teaching Experience}\label{teaching-experience}}

\begin{cventries}
    \cventry{Computational Genomics module}{University of Zagreb, Faculty of Science, Bioinformatics Group}{Zagreb}{2017 - 2023}{\begin{cvitems}
\item Annual participation as a lecturer in 'Algorithms and programming' course for 2nd year master students - teaching R programming, designing and grading practical homework assignments, designing hackathon sessions and providing consultations to approx. 10 students per year
\end{cvitems}}
    \cventry{Summer School in Bioinformatics}{Mediterranean Institute for Life Sciences}{Split}{2022}{\begin{cvitems}
\item Presenting a practical, hands-on lecture on basics of RNA-seq data analysis for 30 participants
\item Designing and teaching a 3 day interactive workshop - tutoring 6 participants how to find and download RNA-seq data from published papers, do the quality check, trim adapters and map reads to the genome using Galaxy framework, with subsequent exploratory and differential expression analysis done in R
\end{cvitems}}
\end{cventries}

\hypertarget{peer-reviewed-publications}{%
\section{Peer-reviewed Publications}\label{peer-reviewed-publications}}

\footnotesize

\hypertarget{bibliography}{}
\leavevmode\vadjust pre{\hypertarget{ref-kataruka2022physiologically}{}}%
Kataruka, S., Kinterova, V., \textbf{Horvat, F.}, Kulmann, M. I. R.,
Kanka, J., \& Svoboda, P. (2022). Physiologically relevant miRNAs in
mammalian oocytes are rare and highly abundant. \emph{EMBO Reports},
\emph{23}(2), e53514.

\leavevmode\vadjust pre{\hypertarget{ref-petrzilek2022novo}{}}%
Petrzilek, J., Pasulka, J., Malik, R., \textbf{Horvat, F.}, Kataruka,
S., Fulka, H., \& Svoboda, P. (2022). De novo emergence, existence, and
demise of a protein-coding gene in murids. \emph{BMC Biology},
\emph{20}(1), 1--14.

\leavevmode\vadjust pre{\hypertarget{ref-zapletal2022structural}{}}%
Zapletal, D., Taborska, E., Pasulka, J., Malik, R., Kubicek, K., Zanova,
M., Much, C., Sebesta, M., Buccheri, V., \textbf{Horvat, F.}, et al.
(2022). Structural and functional basis of mammalian microRNA biogenesis
by dicer. \emph{Molecular Cell}, \emph{82}(21), 4064--4079.

\leavevmode\vadjust pre{\hypertarget{ref-loubalova2021formation}{}}%
Loubalova, Z., Fulka, H., \textbf{Horvat, F.}, Pasulka, J., Malik, R.,
Hirose, M., Ogura, A., \& Svoboda, P. (2021). Formation of spermatogonia
and fertile oocytes in golden hamsters requires piRNAs. \emph{Nature
Cell Biology}, \emph{23}(9), 992--1001.

\leavevmode\vadjust pre{\hypertarget{ref-ganesh2020most}{}}%
Ganesh, S., \textbf{Horvat, F.}, Drutovic, D., Efenberkova, M., Pinkas,
D., Jindrova, A., Pasulka, J., Iyyappan, R., Malik, R., Susor, A., et
al. (2020). The most abundant maternal lncRNA Sirena1 acts
post-transcriptionally and impacts mitochondrial distribution.
\emph{Nucleic Acids Research}, \emph{48}(6), 3211--3227.

\leavevmode\vadjust pre{\hypertarget{ref-kataruka2020microrna}{}}%
Kataruka, S., Modrak, M., Kinterova, V., Malik, R., Zeitler, D. M.,
\textbf{Horvat, F.}, Kanka, J., Meister, G., \& Svoboda, P. (2020).
MicroRNA dilution during oocyte growth disables the microRNA pathway in
mammalian oocytes. \emph{Nucleic Acids Research}, \emph{48}(14),
8050--8062.

\leavevmode\vadjust pre{\hypertarget{ref-demeter2019main}{}}%
Demeter, T., Vaskovicova, M., Malik, R., \textbf{Horvat, F.}, Pasulka,
J., Svobodova, E., Flemr, M., \& Svoboda, P. (2019). Main constraints
for RNAi induced by expressed long dsRNA in mouse cells. \emph{Life
Science Alliance}, \emph{2}(1).

\leavevmode\vadjust pre{\hypertarget{ref-taborska2019restricted}{}}%
Taborska, E., Pasulka, J., Malik, R., \textbf{Horvat, F.}, Jenickova,
I., Jelić Matošević, Z., \& Svoboda, P. (2019). Restricted and
non-essential redundancy of RNAi and piRNA pathways in mouse oocytes.
\emph{PLoS Genetics}, \emph{15}(12), e1008261.

\leavevmode\vadjust pre{\hypertarget{ref-horvat2018role}{}}%
\textbf{Horvat, F.}, Fulka, H., Jankele, R., Malik, R., Jun, M.,
Solcova, K., Sedlacek, R., Vlahovicek, K., Schultz, R. M., \& Svoboda,
P. (2018). Role of Cnot6l in maternal mRNA turnover. \emph{Life Science
Alliance}, \emph{1}(4).

\leavevmode\vadjust pre{\hypertarget{ref-franke2017long}{}}%
Franke, V., Ganesh, S., Karlic, R., Malik, R., Pasulka, J.,
\textbf{Horvat, F.}, Kuzman, M., Fulka, H., Cernohorska, M., Urbanova,
J., et al. (2017). Long terminal repeats power evolution of genes and
gene expression programs in mammalian oocytes and zygotes. \emph{Genome
Research}, \emph{27}(8), 1384--1394.

\hypertarget{book-chapters}{%
\section{Book Chapters}\label{book-chapters}}

\hypertarget{bibliography}{}
\leavevmode\vadjust pre{\hypertarget{ref-michal_cognitive_2021}{}}%
\textbf{Horvat, F.}, \& Svoboda, P. (2019). Microarray-based
transcriptomics of early mammalian development. In \emph{Advances in
disease models}. OPTIO CZ.



\end{document}
